test_runner/regress/test_logical_replication.py

```diff
@@ -203,6 +203,61 @@ def slot_removed(ep):
     wait_until(number_of_iterations=10, interval=2, func=partial(slot_removed, endpoint))
 
 
+def test_subscriber_before_safekeeper_sync(neon_simple_env: NeonEnv, vanilla_pg):
```

Comment:
Test name is not clear, smth like test_lr_with_sk_wal_sending_disabled would be better; also would be great to have general comment on the test what it does.

Reply:
Changed name adn added comment